ui/src/app/cron-workflows/components/schedule.tsx

```diff
@@ -0,0 +1,42 @@
+import React = require('react');
+import {SuccessIcon, WarningIcon} from '../../shared/components/fa-icons';
+
+const x = require('cronstrue');
+
+/*
+    https://github.com/bradymholt/cRonstrue
+    vs
+    https://github.com/robfig/cron
+
+    I think we must assume that these libraries (or any two libraries) will never be exactly the same and accept that
+    sometime it'll not work as expected. Therefore, we must let the user know about this.
+ */
+
+export const Schedule = ({schedule}: {schedule: string}) => {
+    try {
+        const pretty = x.toString(schedule);
+        return (
+            <span title={pretty}>
+                {schedule} <small>({pretty})</small>
```

Comment:
Not sure if parenthesis is my favorite way to segment them. Perhaps
```suggestion
                <code>{schedule}</code> {pretty}
```
?